source/unified-test-format/tests/invalid/expectedTracingSpans-additionalProperties.yml

```diff
@@ -0,0 +1,16 @@
+description: "expectedTracingSpans-additionalProperties"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+
+tests:
+  - description: "additional property foo not allowed in expectTracingMessages"
+    operations: []
+    expectTracingMessages:
+      client: *client0
+      ignoreExtraSpans: false
+      spans: [{}]
```

Comment:
`spanComponent` requires `name` and `tags` fields, so this file might actually fail validation due to _that_ instead of the extra `foo` field below. For any invalid syntax files, make sure that there is exactly one syntax variation that causes a validation failure.